configuration/jsr223.md

```diff
@@ -13,46 +13,44 @@ title: JSR223 Scripting
 
 ## Overview
 
-[JSR223](https://docs.oracle.com/javase/6/docs/technotes/guides/scripting/) ([spec](https://jcp.org/aboutJava/communityprocess/pr/jsr223/index.html)) 
-is a standard scripting API for Java Virtual Machine (JVM) 
-[languages](https://en.wikipedia.org/wiki/List_of_JVM_languages). 
+[JSR223](https://docs.oracle.com/javase/6/docs/technotes/guides/scripting/) ([spec](https://jcp.org/aboutJava/communityprocess/pr/jsr223/index.html)) is a standard scripting API for Java Virtual Machine (JVM) [languages](https://en.wikipedia.org/wiki/List_of_JVM_languages). 
 The JVM languages provide varying levels of support for the JSR223 API and interoperability with the Java runtime. 
 Currently the following languages are known to work well for openHAB scripting: 
-[**Oracle Nashorn**](http://www.oracle.com/technetwork/articles/java/jf14-nashorn-2126515.html) (Javascript bundled with Java),
-[**Jython**](http://www.jython.org/) (Python on the JVM) and
+[**Jython**](https://jython.github.io/) (Python on the JVM), 
+[**Nashorn Javascript**](http://www.oracle.com/technetwork/articles/java/jf14-nashorn-2126515.html) (ECMAScript implementation included in JDK8 through 10, [deprecated in 11](https://openjdk.java.net/jeps/335)), and 
 [**Apache Groovy**](http://www.groovy-lang.org/) (JVM scripting language).
 
-Although JSR223 scripts can be used as a general-purpose extension language for openHAB, 
-the most common usage will be for defining openHAB rules.
+Although JSR223 scripts can be used as a general-purpose extension language for openHAB, it is most commonly used for the creation of rules, and within scripted Actions or Conditions. 
+Currently, openHAB allows JSR223 scripting to access all packages, which may not be included in the official APIs. 
+This provides great flexibility for the users of JSR223, but is also 'use at your own risk', since changes outside of the offical APIs occur frequently, and are not considered to be 'breaking changes'.
```

Comment:
```suggestion
This provides great flexibility for the users of JSR223, but is also *use at your own risk*, since changes outside of the offical APIs occur frequently, and are not considered to be *breaking changes*.
```